laa_court_data_api_app/routers/defendants.py

```diff
@@ -72,3 +71,16 @@ async def get_defendants(urn: str | None = None,
         case _:
             logging.error("Prosecution_Case_Endpoint_Error_Returning")
             return Response(status_code=424)
+
+
+def map_defendants(prosecution_case_results: ProsecutionCasesResults) -> list[DefendantSummary]:
+    response_list = []
+    for results in prosecution_case_results.results:
```

Comment:
could this be result not results
```suggestion
    for result in prosecution_case_results.results:
```